docs/sources/file-input.mdx

```diff
@@ -0,0 +1,166 @@
+---
+sidebar_position: 3
+slug: /file-input
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+import UppyCdnExample from '/src/components/UppyCdnExample';
+
+# File input
+
+The `@uppy/file-input` plugin is the most barebones UI for selecting files — it
+shows a single button that, when clicked, opens up the browser’s file selector.
+
+## When should I use it?
+
+When you want users to select files from their local machine with a minimal UI.
+
+## Install
+
+<Tabs>
+  <TabItem value="npm" label="NPM" default>
+
+```shell
+npm install @uppy/file-input
+```
+
+  </TabItem>
+
+  <TabItem value="yarn" label="Yarn">
+
+```shell
+yarn add @uppy/file-input
+```
+
+  </TabItem>
+
+  <TabItem value="cdn" label="CDN">
+    <UppyCdnExample>
+      {`
+        import { Uppy, FileInput } from "{{UPPY_JS_URL}}"
+        const uppy = new Uppy()
+        uppy.use(FileInput, { target: document.body })
+      `}
+    </UppyCdnExample>
+  </TabItem>
+</Tabs>
+
+## Use
+
+```js showLineNumbers
+import Uppy from '@uppy/core';
+import FileInput from '@uppy/file-input';
+
+import '@uppy/core/dist/style.min.css';
+import '@uppy/file-input/dist/style.css';
+
+new Uppy().use(FileInput, { target: '#uppy-file-input' });
+```
+
+:::note
+
+The `@uppy/file-input` plugin includes some basic styles for use with the
+[`pretty`](#pretty-true) option, like shown in the
+[example](/examples/xhrupload). You can also choose not to use it and provide
+your own styles instead.
+
+Import general Core styles from `@uppy/core/dist/style.css` first, then add the
+File Input styles from `@uppy/file-input/dist/style.css`. A minified version is
+also available as `style.min.css` at the same path. The way to do import depends
+on your build system.
+
+:::
+
+## API
+
+### Options
+
+#### `id`
+
+A unique identifier for this plugin (`string`, default: `'FileInput'`).
+
+#### `title`
+
+Configures the title / name shown in the UI, for instance, on Dashboard tabs
+(`string`, default: `'File Input'`).
+
+#### `target`
+
+DOM element, CSS selector, or plugin to place the audio into (`string` or
+`Element`, default: `null`).
+
+#### `pretty`
+
+When true, display a styled button (see [example](/examples/xhrupload)) that,
+when clicked, opens the file selector UI. When false, a plain old browser
+`<input type="file">` element is shown (`boolean`, default: `true`).
+
+#### `inputName`
+
+The `name` attribute for the `<input type="file">` element (`string`, default:
+`'files[]'`).
+
+#### `locale`
+
+```js
+export default {
+	strings: {
+		// The same key is used for the same purpose by @uppy/robodog's `form()` API, but our
+		// locale pack scripts can't access it in Robodog. If it is updated here, it should
+		// also be updated there!
+		chooseFiles: 'Choose files',
+	},
+};
+```
+
+## Custom file inpt
```

Comment:
This doesn't really capture the intricacies of building your own UI with Uppy, which we are planning to document with a [guide](https://uppy.io/docs/guides/building-your-own-ui-with-uppy/). Perhaps we can skip this whole section and describe the alternative of building from scratch in "when should I use it".

Reply:
Or we should keep that section until the PR adding said guide removes it in replacement for a link. wdyt?